doc/integrator/docker.rst

```diff
@@ -38,7 +38,7 @@ Run the developer composition
 
    docker-compose up -d
 
-You can then access your application with http://localhost:8480/
+You can then access your application with http://172.17.0.1:8480/
```

Comment:
This higly depends on plateform and docker packaging. Be careful. You should warn user

Reply:
After a more careful reading. Why are you changing localhost with the docker bridge ip ? https://github.com/camptocamp/c2cgeoportal/blob/master/geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose-lib.yaml_tmpl#L217 it seems that we can access the front from localhost since port are exposed.